src/process/ProcessTree.c

```diff
@@ -346,9 +346,14 @@ pid_t ProcessTree_findProcess(Service_T s) {
                 pid_t pid = Util_getPid(s->path);
                 if (pid > 0) {
                         errno = 0;
-                        if (getpgid(pid) > -1 || errno == EPERM)
+                        if (getpgid(pid) > -1 || errno == EPERM){
+                                if (errno == EPERM) {
+                                        LogError("getpgid returned errno == EPERM for pid %d", pid);
+                                }
                                 return pid;
-                        DEBUG("'%s' process test failed [pid=%d] -- %s\n", s->name, pid, STRERROR);
+                        } else {
+                                DEBUG("'%s' process test failed [pid=%d] -- %s\n", s->name, pid, STRERROR);
```

Comment:
Just tested, and don't think we should. Log gets pretty spammy at e.g. upgrade, or if you have a cyclicly dead service (accidentally broke sprout with some mismatched packages). Monit log ends up with a few hundred lines just from initial install, and then 30+ for every restart attempt on the failing service. 
Going to leave it as is unless you shout at some later date